lldb/test/Shell/Breakpoint/breakpoint-command.test

```diff
@@ -1,7 +1,7 @@
 # REQUIRES: lua || python
 
 # RUN: %build %p/Inputs/dummy-target.c -o %t.out
-# RUN: %lldb %t.out -o 'b main' -o 'break command add 1 -o "script print(95000 + 126)"' -o 'r' | FileCheck %s
+# RUN: %lldb %t.out -o 'b main' -o 'break command add 1 -o "script print(\"A R4Nd0m\" + \" Str1ng\")"' -o 'r' | FileCheck %s
```

Comment:
Interesting, is there a specific reason for doing string concatenation here? Or just keeping the `+` from the previous version?

Reply:
Without string concatenation, the test detects the string in the script command, so that it is still found multiple times.